Adamant/Modules/Chat/View/ChatViewController.swift

```diff
@@ -189,11 +194,29 @@ final class ChatViewController: MessagesViewController {
         super.collectionView(collectionView, willDisplay: cell, forItemAt: indexPath)
     }
     
+    override func scrollViewDidEndDragging(_: UIScrollView, willDecelerate _: Bool) {
```

Comment:
Are you sure it's correct? At this point scroll can still have momentum and can actually stop a bit later